src/hotspot/cpu/aarch64/aarch64.ad

```diff
@@ -2554,6 +2550,25 @@ bool Matcher::is_spillable_arg(int reg)
   return can_be_java_arg(reg);
 }
 
+uint Matcher::int_pressure_limit()
+{
+  // JDK-8183543: Take the number of available registers as int register
+  // pressure threshold, the jtreg test:
+  // test/hotspot/jtreg/compiler/regalloc/TestC2IntPressure.java
+  // failed due to C2 compilation failure with
+  // "COMPILE SKIPPED: failed spill-split-recycle sanity check".
+  // Lowering the default value to (_NO_SPECIAL_REG32_mask.Size() minus 1)
+  // fixes this bug. See JDK-8183543.
+  uint int_pressure_threshold = _NO_SPECIAL_REG32_mask.Size() - 1;
```

Comment:
This is not explanation. Comment should explain why with -1 the issue is fixed. (Repeating because it could be hidden in previous conversation)

Reply:
I add more details in comments. Please let me know if I explain clearly.